xdsl/transforms/csl_stencil_bufferize.py

```diff
@@ -94,6 +94,9 @@ def match_and_rewrite(self, op: csl_stencil.ApplyOp, rewriter: PatternRewriter,
         # convert args
         buf_args: list[SSAValue] = []
         to_memrefs: list[Operation] = [buf_iter_arg := to_memref_op(op.accumulator)]
+        op.accumulator.replace_by_if(
+            buf_iter_arg.memref, lambda use: use.operation != buf_iter_arg
+        )
```

Comment:
Allowing the `bufferization.to_memref` (buf_iter_arg) to be used by further apply ops, in case there is more than one.